cloud-computing/src/app.py

```diff
@@ -90,6 +90,18 @@ def handle_requester_secrets():
     return iexec_requester_secrets
 
 
+def write_stderr():
```

Comment:
I don't see no `write_stderr()` call in main or elsewhere, is it expected?

Reply:
Well... No. Thanks. :upside_down_face: